linalg/densemat.hpp

```diff
@@ -1283,6 +1283,22 @@ void BatchLUFactor(DenseTensor &Mlu, Array<int> &P, const double TOL = 0.0);
 void BatchLUSolve(const DenseTensor &Mlu, const Array<int> &P, Vector &X);
 
 
+void BatchLUFactor(const int m, const int len,
+                   mfem::Vector &A, mfem::Array<int> &P);
+
+void BatchLUSolve(mfem::Vector &Minv, int m, int NE, mfem::Array<int> &P,
+                  mfem::Vector &X);
+
+void BatchInverseMatrix(const Vector &LU,
+                        const int m,
+                        const int NE,
+                        const Array<int> &P,
+                        Vector &INV);
+
+void BatchInverseMatrix(const DenseTensor &LU,
+                        const Array<int> &P,
+                        DenseTensor &INV);
```

Comment:
It looks like there are existing `BatchLUFactor` and BatchLUSolve` methods, can we combine those and these?

Reply:
For generality please consider using `len`/`length` instead of `NE`